test/dbus/lvmdbustest.py

```diff
@@ -85,15 +121,11 @@ def get_objects():
 	for object_path, v in objects.items():
 		proxy = ClientProxy(bus, object_path, v)
 		for interface, prop in v.items():
-			if interface == PV_INT:
-				# If we have a list of PVs to use, lets only use those in
-				# the list
-				# noinspection PyUnresolvedReferences
-				if pv_device_list and not (proxy.Pv.Name in pv_device_list):
-					continue
 			rc[interface].append(proxy)
```

Comment:
``prop`` is not being use here, it would be better to just iterate over ``v.keys()``.